incubator/esbuild-bundle-analyzer/src/webpackStats.ts

```diff
@@ -0,0 +1,148 @@
+import fs from "fs";
+import * as path from "path";
+import { filesToSkip } from "./constants.js";
+import { generateGraph, getWhyFileInBundle } from "./duplicates.js";
+import type { Metafile } from "./metafile.js";
+import type {
+  Graph,
+  StatsModuleIssuer,
+  StatsModuleReason,
+  WebpackStats,
+} from "./types.js";
+import { info } from "@rnx-kit/console";
+
+// TODO: Update to be more generic and configurable to it supports
+// more than just lib and src folders
+function getLine(filePath: string, keyword: string): string {
+  try {
+    const file = fs.readFileSync(filePath, "utf-8");
+    const lineNumber = file.split("\n").findIndex((line) => {
+      const result = line.includes(keyword);
+
+      if (!result && keyword.includes("/src/index.ts")) {
+        return line.includes(keyword.split("/src/index.ts")[0]);
+      }
+
+      if (!result && line.includes("/lib/")) {
+        return line
+          .replace("/lib/", "/src/")
+          .replace(".js", ".ts")
+          .includes(keyword);
+      }
+
+      return result;
+    });
+
+    return lineNumber !== -1 ? (lineNumber + 1).toString() : "";
```

Comment:
In general, it's better compare against 0:
```suggestion
    return lineNumber >= 0 ? (lineNumber + 1).toString() : "";
```